kmip/pie/objects.py

```diff
@@ -446,12 +473,13 @@ class PrivateKey(CryptographicObject):
         cryptographic_algorithm: The type of algorithm for the PrivateKey.
         cryptographic_length: The length in bits of the PrivateKey.
         value: The bytes of the PrivateKey.
+        format_type: The format of the key value.
         cryptographic_usage_masks: The list of usage mask flags for PrivateKey
             application.
         names: The list of string names of the PrivateKey.
     """
 
-    def __init__(self, algorithm, length, value, masks=None,
+    def __init__(self, algorithm, length, value, format_type, masks=None,
```

Comment:
Similar comment on format_type as for public keys and adding todo for checking format_type and value. I think raw, pkcs1, and pkcs8 for now. I know there is ECPrivateKey, but I don't know much about that one.